core/domain/feedback_jobs_continuous.py

```diff
@@ -145,6 +145,38 @@ def get_thread_analytics(cls, exploration_id):
         return feedback_domain.FeedbackAnalytics(
             exploration_id, num_open_threads, num_total_threads)
 
+    @classmethod
+    def get_thread_analytics_multi(cls, exploration_ids):
+        realtime_model_ids = cls.get_multi_active_realtime_layer_ids(
+            exploration_ids)
+        realtime_models = cls._get_realtime_datastore_class().get_multi(
+            realtime_model_ids)
+        feedback_thread_analytics_models = (
+            feedback_models.FeedbackAnalyticsModel.get_multi(
+                exploration_ids))
+
+        num_open_threads_list = []
+        num_total_threads_list = []
+        for i in range(len(exploration_ids)):
+            num_open_threads = 0
+            num_total_threads = 0
+            if realtime_models[i] is not None:
```

Comment:
The realtime model is actually independent of the MR job. It just records stuff that's come in on the fly.
The feedback analytics model is only created when the MR job runs and there is something for it to record.
Your test should call this method, i.e.: get_thread_analytics_multi(). All the tests you pointed to are for get_thread_analytics(). You need new tests for this method.

Here is a way you can set it up:

Create two explorations.
Call get_thread_analytics_multi() for both and check that it returns the right thing.

Submit feedback to the first exploration.
Call get_thread_analytics_multi() for both explorations and check that it returns the right thing.

Run the feedback aggregator job.
Call get_thread_analytics_multi() for both explorations and check that it returns the right thing.